changelog/unreleased/pull-5054

```diff
@@ -0,0 +1,18 @@
+Enhancement: Add --compress-zip flag to dump command to compress archive
+
+# Describe the problem in the past tense, the new behavior in the present
+# tense. Mention the affected commands, backends, operating systems, etc.
+# Focus on user-facing behavior, not the implementation.
+# Use "Restic now ..." instead of "We have changed ...".
```

Comment:
These comments can be excluded from the changelog.

Reply:
Oh, must have slipped by...